praw/models/reddit/subreddit.py

```diff
@@ -1615,6 +1648,38 @@ def create(self, subject, body, recipient, author_hidden=False):
         return self.subreddit._reddit.post(API_PATH['modmail_conversations'],
                                            data=data)
 
+    def subreddits(self):
+        """Yield subreddits using the new modmail that the user moderates.
+
+        Example:
+
+        .. code:: python
+
+           subreddits = reddit.subreddit('all').modmail.subreddits()
+
+        """
+        response = self.subreddit._reddit.get(API_PATH['modmail_subreddits'])
+        for value in response['subreddits'].values():
+            subreddit = self.subreddit._reddit.subreddit(value['display_name'])
+            subreddit.last_updated = value['lastUpdated']
+            yield subreddit
```

Comment:
Since this is a get request, I think it was a great choice to make this a generator.